package.json

```diff
@@ -95,6 +96,7 @@
         "@unibeautify/beautifier-prettydiff": "^0.5.3",
         "@unibeautify/beautifier-sqlformat": "^0.0.1",
         "cosmiconfig": "^5.0.5",
+        "csscomb": "^4.2.0",
```

Comment:
New information: https://github.com/Unibeautify/beautifier-csscomb/pull/12#issuecomment-425911302

Apologize for the confusion. I need to stop reviewing PRs so late at night 😜 .